spec/lucky/action_redirect_spec.cr

```diff
@@ -144,6 +144,16 @@ describe Lucky::Action do
     flash = Lucky::FlashStore.from_session(response.context.session)
     flash.success.should eq("Keep me!")
   end
+
+  it "closes the response after a redirect" do
+    action = RedirectAction.new(build_context, params)
+    action.redirect to: "/somewhere"
+    should_redirect(action, to: "/somewhere", status: 302)
+
+    expect_raises(IO::Error) do
+      action.redirect to: "/somewhere-else"
+    end
```

Comment:
This is a weird test methodology. It feels like it might lead to a false negative, where this test passes but for the wrong reason. Is there a hook to ask the action if the response is closed somehow?

Reply:
I could do `action.context.response.closed?.should eq(true)`. :man_shrugging: